packages/ssi-service/src/feedback-panel/feedback.js

```diff
@@ -0,0 +1,293 @@
+import html from 'html-template-tag';
+import { GraphQLClient } from 'graphql-request';
+import styles from './feedback.css';
+
+window.customElements.define('op-feedback', class extends HTMLElement {
+  constructor () {
+    super();
+
+    this.graphqlClient = new GraphQLClient('https://localhost:8080/graphql', {
```

Comment:
If possible use the environment variables.

Reply:
Okay. Will probably add this in the build script.